files/bootstrap.sh

```diff
@@ -191,6 +191,8 @@ if vercmp "$KUBELET_VERSION" lt "1.27.0"; then
 fi
 MOUNT_BPF_FS="${MOUNT_BPF_FS:-$DEFAULT_MOUNT_BPF_FS}"
 
+KUBELET_ARGS=""
```

Comment:
Why is this needed?

Reply:
I was misreading line numbers while jumping around this file -- it's not needed, for now.